awscli/testutils.py

```diff
@@ -89,6 +89,21 @@ def decorator(func):
     return decorator
 
 
+def skip_if_not_windows(reason):
```

Comment:
Do we already have precedent for this name?

Reply:
Not really. I could name it to `skip_if_macos_or_linux` if that is better? I know there is this PR that adds a new skip decorator for macos: https://github.com/aws/aws-cli/pull/4664 but there is not a precedent for a group of operating systems.

Reply:
I mostly dislike the sort of double-negative 'skip if not'. Something like `if_windows`, `on_windows`, `only_on_windows`.